### ISSUE TITLE
Open source font generation

### ISSUE THREAD

**Issue by max** [google/material-design-icons#139]
It would be neat if you open sourced how you generate the icon font (including the configuration). Makes it easier to extend the library with additional (proprietary) icons.

**Comment by shyndman** [comment]
We would love to share our icon generation process at some point, but we're not quite there yet. In the meantime, I recommend using a tool like [Fontello](http://fontello.com/) to create a custom cut from a variety of sources.

**Comment by Angelmmiguel** [comment]
:+1:

**Comment by EvanEdwards** [comment]
Agreed, this would be wonderful.  While Fontello is nice, a solid default toolchain for maintaining a project with them would be even better.  Good to hear it's at least a plausible idea.

**Comment by bmcleod** [comment]
I have a requirement to extend the existing material icon font with new glyphs. I've looked in to the gulp tasks and the files it generates, but need further documentation. Is the icon generation process something you are considering to document at this time?
